K8s-deployment/Charts/databroker/example-secrets/secrets/init-config.json

```diff
@@ -324,6 +324,12 @@
       "queue_vhost": "IUDX-INTERNAL",
       "queue_binding_exchange": "auditing",
       "queue_binding_key": "#"
+    },
+    {
+      "queue_name": "subscriptions-monitoring",
```

Comment:
- Need to also add extra permissions to auditing user as follows : 
![Screenshot_2024-05-10_10-43-03](https://github.com/datakaveri/iudx-deployment/assets/29863835/e4856787-74bd-4c10-9b09-b423b39a1d5c)
- Need to add dmp-apd user ``dx-dmp-apd-user`` with permissions similar to dx-di-user.